src/macros.rs

```diff
@@ -329,51 +325,54 @@ macro_rules! service {
             rpc $fn_name( $( $arg : $in_ ),* ) -> $out | $error;
         }
     };
-    // Pattern for when all return types have been expanded
+// Pattern for when all return types have been expanded
```

Comment:
Yeah let's leave them. I tried to keep them nice for a while, which was a pretty frustrating experience.